client/components/sticky-panel/test/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @format
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import ReactDom from 'react-dom';
+
+import { calculateOffset, getBlockStyle, getDimensions, getDimensionUpdates } from '..';
+
+beforeAll( () => {
+	const header = {
+		getBoundingClientRect: () => ( { height: 123 } ),
+	};
+	jest
+		.spyOn( document, 'getElementById' )
+		.mockImplementation( id => ( id === 'header' ? header : null ) );
+	jest.spyOn( ReactDom, 'findDOMNode' ).mockImplementation( node => node );
+} );
+
+describe( 'calculateOffset', () => {
+	test( 'returns the header height', () => {
+		expect( calculateOffset() ).toEqual( 123 );
+	} );
+} );
+
+describe( 'getBlockStyle', () => {
+	test( 'returns a valid style if sticky', () => {
+		const state = { isSticky: true, blockWidth: 200 };
+
+		expect( getBlockStyle( state ) ).toEqual( { top: 123, width: 200 } );
+	} );
+
+	test( 'returns undefined if not sticky', () => {
+		const state = { isSticky: false, blockWidth: 200 };
+
+		expect( getBlockStyle( state ) ).toBeUndefined;
```

Comment:
This should be a function call:
```suggestion
		expect( getBlockStyle( state ) ).toBeUndefined();
```

Reply:
Very sharp 👀 you have @sirreal! And thanks for demo of the new GitHub feature :+1: